1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,36 @@
+// ## Exercise
+
+// - Iterate through the list of books.
+// - For each book, create a `<p>` element with the book title and author.
+// - Add an `<img>` to each book that links to a URL of the book cover.
+// - Use a `<ul>` and `<li>` to display the books and append it to the page.
+// - Change the style of the book depending on whether you have read it (green) or not (red). Hint: try adding some CSS classes to styles.css for these styles.
+// - All of your HTML should go inside the `<div>` with the id 
+
+
+
+
+
 function readingList(books) {
   // Write your code here...
+  let content = document.querySelector('#content');
+  let unorderedList = document.createElement('ul')
+  content.appendChild(unorderedList);
+  books.forEach(book => {
+    let list = document.createElement('li');
```

Comment:
Would be nice to add some classes to all the items so we can have some better styling. (Such as max sizes for the images)